src/drive_cpp/include/drive_cpp/TalonDriveController.hpp

```diff
@@ -20,7 +20,7 @@ class TalonDriveController : public rclcpp::Node {
   /**
    * @brief Construct a new TalonDriveController object.
    */
-  TalonDriveController();
+  explicit TalonDriveController(const rclcpp::NodeOptions& options);
```

Comment:
@ConnorNeed  @TheWalrus368  This PR will change the drive_cpp to component setup. Need to double check this is ok and looks good.